Lombiq.NodeJs.Extensions.Samples/.stylelintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
There are created for the NuGet project too:

![image](https://user-images.githubusercontent.com/1976647/164082372-331235b9-63d5-4aab-ba29-bc2622b7a06c.png)

(The pnpm-lock in the base project shows a change without actually changing; I suspect it has something to do with EOL - https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/65 - but Git doesn't show a change.)

Reply:
Should I just check these in? I didn't do that on purpose, because I thought they would be part of the learning effect.

I don't have that change of the _pnpm-lock_ file in the `Samples` project.

Reply:
Yes, invoking a build shouldn't cause the clone to become dirty.